milvus/ClientProxy.go

```diff
@@ -0,0 +1,224 @@
+package milvus
+
+import (
+	"flag"
+	pb "github.com/milvus-io/milvus-sdk-go/milvus/grpc/gen"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/testdata"
+	"log"
+)
+
+var (
+	tls                = flag.Bool("tls", false, "Connection uses TLS if true, else plain TCP")
+	caFile             = flag.String("ca_file", "", "The file containing the CA root cert file")
```

Comment:
gprc need tls / ca?

Reply:
tls / ca is an optional but not necessary function for grpc. I put it here just as a template, in fact, I didn't use it in our project.